UM/Application.py

```diff
@@ -38,6 +38,11 @@
 
 @signalemitter
 class Application:
+    class ShutdownSeverity:
+        HardCrash = "hard_crash"
+        CrashHandler = "crash_handler"
```

Comment:
Wouldn't the description "handled crash" be a bit more readable ?

Reply:
This isn't used as a description, it's used as values in the config-file. I'm not sure that can handle a space.

The CrashHandler is a specific part of our code, although on the other hand I do mean 'handled crash' in a more general sense (even though that's the only way we can handle crashes).

Reply:
Oh I didn't mean to remove the space of course..

But as you way, "handled crash" is a more generic description, whenever there is class called `CrashHandler` or a `SometingBadIsHappeningHandler` or whatever..

Reply:
True, ok, I'll change it.